src/app/modules/core/advertisement/components/latest/latest-mini-card/latest-mini-card.component.ts

```diff
@@ -1,15 +1,16 @@
-import { Component, OnInit } from '@angular/core'
-import { PROPERTIES_MOCK_DATA } from '@shared/models/mock-data/data'
+import { Component, Input, OnInit } from '@angular/core'
+
+import { AdvertisementService } from './../../../advertisement.service'
 
 @Component({
   selector: 'cav-latest-mini-card',
   templateUrl: './latest-mini-card.component.html',
   styleUrls: ['./latest-mini-card.component.scss'],
 })
 export class LatestMiniCardComponent implements OnInit {
-  constructor() {}
+  constructor(private advService: AdvertisementService) {}
 
-  properties = PROPERTIES_MOCK_DATA
+  @Input() latest
```

Comment:
Qui andrebbe quindi cambiato pure il nome in `property` e il tipo sarà `IProperty`